add style for table formatting to docs contribution (#15612)
apache/druid: docs/development/docs-contribute.md

HUMAN_READABLE_BINARY_BYTE_FORMAT(value, \[precision])
:::

#### Markdown table format

> When editing or adding tables, do not include extra characters to "prettify" the table format within the markdown source.

Reviewer: ```suggestion
When editing or adding tables, do not include extra characters to "prettify" the table format within the Markdown source.
```

Some code editors may format tables by default.
See the developer [style guide](https://github.com/apache/druid/blob/master/dev/style-conventions.md) for more information.

:::tip

**Correct**

> ```

Reviewer 2: ```suggestion
```markdown
```

> For an example of how to disable it, see the [IntelliJ formatting file for Druid](https://github.com/apache/druid/blob/master/dev/druid_intellij_formatting.xml#L78).

Reviewer 3: Or it'd be better to import the code style file - https://github.com/apache/druid/blob/master/dev/style-conventions.md

> For an example of how to disable it, see the [IntelliJ formatting file for Druid](https://github.com/apache/druid/blob/master/dev/druid_intellij_formatting.xml#L78).

Author: ```suggestion
See the [developer Style Guide for more information](https://github.com/apache/druid/blob/master/dev/style-conventions.md).
```
Thanks for the suggestion @abhishekrb19 . See revision.

| Column 1 | Column 2 | Column 3 |
| --- | --- | --- |
| value 1 | val 2 | a-very-long-value 3 |
```

**Incorrect**

> ```

Reviewer 2: ```suggestion
```markdown
```
